### PR TITLE
update hp procurve show_vlans and show_mac-address

### PR DESCRIPTION
**Pull request by dainok**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT

hp_procurve_show_mac-address.textfsm

##### SUMMARY

Adding a record without the vlan field for very old HP Procurve switches.
Also providing additional test for "show vlans" command.

### CODE REVIEW

**Comment by dainok** [comment]
Ok, I'm asking for help: tox is failing on files I didn't updated. I tested on a previous brach (3 weeks ago) and tox is working fine. What I can do?

**Comment by dainok** [comment]
@jvanderaa is there anything I can do for help? On my side everything seems ok, but maybe I missed something. Let me know.

**Comment by jvanderaa** [comment]
The only thing I am needing to put some thought around is why the change on the space search from `+` to `*`.

File: tests/cisco_nxos/show_ip_interface/cisco_nxos_show_ip_interface1.raw [hunk 1]
```diff
@@ -0,0 +1,31 @@
+IP Interface Status for VRF "default"
```
Comment (itdependsnetworks) [inline tests/cisco_nxos/show_ip_interface/cisco_nxos_show_ip_interface1.raw:1]: While not impossible, this does not seem likely like a raw output of the command (based on the other commands). I am going to guess there is something else in your process that is stripping the whitespace before the lines. Can you confirm if that is the case or not?

Comment (dainok) [reply tests/cisco_nxos/show_ip_interface/cisco_nxos_show_ip_interface1.raw:1]: I'm sorry the changes on NXOS files were taken from https://github.com/networktocode/ntc-templates/pull/1046
Initially, testing my commit, I had some issues on Cisco files I didn't changed. Few days later I synced the master repo again and tests completed successfully without other changes. In that moment, probably, I made some errors including the commit from 1046. Let me know if you can still merge or if I should make another PR.
My PR involves HP Procurve files only.

Comment (dainok) [reply tests/cisco_nxos/show_ip_interface/cisco_nxos_show_ip_interface1.raw:1]: @jvanderaa, @itdependsnetworks any thoughts? Should I make another PR?

**Comment by dainok** [comment]
Creating a new and clean one: https://github.com/networktocode/ntc-templates/pull/1108

**Pull request closed without merging**